<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="day_tab">
    <attribute name="title">
     <string>day</string>
    </attribute>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>751</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>房间空调开关次数</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>温控请求统计</string>
      </attribute>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>621</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>当天所需总费用</string>
      </attribute>
      <widget class="QTableView" name="costtable">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>0</y>
         <width>256</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>290</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>总费用：</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="week_tab">
    <attribute name="title">
     <string>week</string>
    </attribute>
    <widget class="QToolBox" name="toolBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>751</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>房间空调开关次数</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>温控请求统计</string>
      </attribute>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>621</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>当月所需总费用</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>280</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>总费用：</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="month_tab">
    <attribute name="title">
     <string>month</string>
    </attribute>
    <widget class="QToolBox" name="toolBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>751</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>房间空调开关次数</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>温控请求统计</string>
      </attribute>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>621</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="label">
       <string>当年所需总费用</string>
      </attribute>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>280</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>总费用：</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
